### PR TITLE
Stream inspector

### PR DESCRIPTION
**Pull request by hariso**

### CODE REVIEW

File: docs/features/stream-inspector.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: 'Stream Inspector'
+sidebar_position: 5
+tag: ['Stream Inspector', 'Connector']
+keywords: ['stream', 'inspector']
+---
+
+Conduit's stream inspector makes it possible to peek at the data as it enters Conduit via source connectors and what the data looks like as it travels to destination connectors. Keep in mind that this feature is about sampling data as it passes through the pipeline not tailing the pipeline.
+
+Stream inspection is available via the Conduit UI and the API. To learn how to use it, please visit our [stream inspector guide](/guides/2022-12-15-stream-inspector).
```
Comment (lovromazgon) [inline docs/features/stream-inspector.mdx:10]: The link is broken, should be `/guides/stream-inspector`

File: guides/2022-12-15-stream-inspector.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+To access the stream inspector through the UI, first navigate to the pipeline which you'd like to inspect. Then, click on the connector in which you're interested. You'll see something similar to this:
+
+![stream inspector pipeline view](/images/stream-inspector-pipeline.png)
+
+Click the "Inspect Stream" button to start inspecting the connector. A new pop-up window will show the records:
+
+![stream inspector show stream](/images/stream-inspector-show-stream.png)
+
+On the "Stream" tab you'll see the latest 10 records. If you switch to the "Single record" view, only the last record will be shown. You can use the "Pause" button to pause the inspector and stop seeing the latest record(s). The ones that are already shown will be kept so you can inspect them more thoroughly.
```
Comment (lovromazgon) [inline guides/2022-12-15-stream-inspector.mdx:21]: > You can use the "Pause" button to pause the inspector and stop seeing the latest record(s).

I had to read it a couple of times to understand what you mean. I propose to change the last part to _"stop fetching new records"_ or _"stop receiving new records"_.

File: guides/2022-12-15-stream-inspector.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## API
+
+To access the stream inspector through the API, you'll need a WebSocket client. [wscat](https://github.com/websockets/wscat) is one of them. The URL on which the inspector is available comes in the following format: `ws://host:port/v1/connectors/<connector ID>/inspect`. For example, if you have Conduit installed locally and with the default settings, you can inspector a connector by running the following command:
```
Comment (lovromazgon) [inline guides/2022-12-15-stream-inspector.mdx:25]:
```suggestion
To access the stream inspector through the API, you'll need a WebSocket client (for example [wscat](https://github.com/websockets/wscat)). The URL on which the inspector is available comes in the following format: `ws://host:port/v1/connectors/<connector ID>/inspect`. For example, if you run Conduit locally with the default settings, you can inspect a connector by running the following command:
```

Comment (lovromazgon) [inline guides/2022-12-15-stream-inspector.mdx:25]: We could also note that the fields `key`, `payload.after`, `payload.before` and `position` can contain raw binary data that's base64 encoded, which can be decoded using `jq` (that's pretty useful IMO).

**Review by lovromazgon — COMMENTED** [review]
Nice work 👍 

I would appreciate it if lines longer than 80 characters would be split into multiple lines so that they are easier to read in an editor 😉

**Comment by hariso** [comment]
> Nice work +1
> 
> I would appreciate it if lines longer than 80 characters would be split into multiple lines so that they are easier to read in an editor wink

Will do... My editor "hides" that from me.:)

**Review by lovromazgon — APPROVED** [review]

**Pull request merged**